command/agent/internal/ctmanager/runner_config.go

```diff
@@ -0,0 +1,148 @@
+package ctmanager
+
+import (
+	"io"
+	"fmt"
+	"strings"
+
+	ctconfig "github.com/hashicorp/consul-template/config"
+	"github.com/hashicorp/go-hclog"
+	"github.com/hashicorp/vault/sdk/helper/pointerutil"
+	ctlogging "github.com/hashicorp/consul-template/logging"
+	"github.com/hashicorp/vault/command/agent/config"
+)
+
+type ManagerConfig struct {
+	AgentConfig *config.Config
+	Namespace   string
```

Comment:
Do we have the ability of configuring per template/env namespace or does it have to be a single one for all requests?

Reply:
seems to be one namespace per vault client (so yes, one)